index.html

```diff
@@ -42,12 +42,15 @@
             </ul>
         </div>
     </section>
+    </ul>
+        </div>
+    </section>
     <div class="headcontainer">
         <div id="overlay">
             <div id="mobilemenu">
                 <div id="option">
                     <div class="login">Login</div>
-                    <a class="link" href="index.html">Home</a>
+                    <a class="link" href="about.html">Home</a>
```

Comment:
- [ ] Please as requested by the previous reviewer, kindly fix the Home page link on the navbar since it redirects to the About page but rather it should redirect to the Home page for better user experience. 😄